docs/manual/manual_angle_control_mechanisms.rst

```diff
@@ -0,0 +1,651 @@
+An invalid set of trajectory commands would be:
+    #.  Command the final velocity to 8 :math:`\frac{rad}{s}`
+    #.  Command the final acceleration to 4 :math:`\frac{rad}{s^2}`
+    #.  Command the duration to 10 s
+    #.  Command the final displacement to 20
+
+Since the duration is given before the displacement, sending this trajectory will have no effect, and your module will not rotate.
+
+Trajectory command parameters are available through the :ref:`Multi Turn Angle Control Client <multi_turn_table>`, 
+and have the names all beginning with *trajectory_*, for example *trajectory_linear_displacement*.
+
+Visualizing Control Trajectories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To help visualize what these parameters do, please refer to the following plots. 
+All figures show module angular displacement on the top, and angular velocity on the bottom. 
+
+.. figure:: ../_static/manual_images/angle_control/trajectory_noaccel_novelo.png
+    :align: center
+    :scale: 60%
+    :alt: Trajectory Command with Final Velocity = 0 :math:`\frac{rad}{s}` and Final Acceleration = 0 :math:`\frac{rad}{s^2}`
+
+    Trajectory Command with Final Velocity = 0 :math:`\frac{rad}{s}` and Final Acceleration = 0 :math:`\frac{rad}{s^2}`
+
+This trajectory sets a final displacement of 150 rad with final velocity of 0 :math:`\frac{rad}{s}`, and final acceleration of 0 :math:`\frac{rad}{s^2}`. 
+The module takes a smooth path to arrive at 150 radians, and once there, has a velocity of 0 :math:`\frac{rad}{s}`, and no acceleration. 
+
+
+.. figure:: ../_static/manual_images/angle_control/trajectory_noaccel_25velo.png
+    :align: center
+    :scale: 60%
+    :alt: Trajectory Command with Final Velocity = 25 :math:`\frac{rad}{s}` and Final Acceleration = 0 :math:`\frac{rad}{s^2}`
+
+    Trajectory Command with Final Velocity = 25 :math:`\frac{rad}{s}` and Final Acceleration = 0 :math:`\frac{rad}{s^2}`
+
+This next trajectory sets the final displacement to 150 rad with final velocity of 25 :math:`\frac{rad}{s}`, and final acceleration of 0 :math:`\frac{rad}{s^2}`. 
+Again, there is a smooth curve in position, but this time a non-zero final velocity. As seen in the plot, 
+we finish the trajectory with a final velocity of 25 :math:`\frac{rad}{s}`, with no acceleration (notice the slope of 0 at the end of the velocity plot).
+
+.. figure:: ../_static/manual_images/angle_control/trajectory_15accel_25velo.png
+    :align: center
+    :scale: 60%
+    :alt: Trajectory Command with Final Velocity = 25 :math:`\frac{rad}{s}` and Final Acceleration = 15 :math:`\frac{rad}{s^2}`
+
+    Trajectory Command with Final Velocity = 25 :math:`\frac{rad}{s}` and Final Acceleration = 15 :math:`\frac{rad}{s^2}`
+
+
+Lastly, a trajectory with a displacement of 150 rad with a final velocity of 25 :math:`\frac{rad}{s}` and a final acceleration of 15 :math:`\frac{rad}{s^2}`. 
+As with all trajectories, the position takes a smooth route to its final displacement, and we once again at a final 
```

Comment:
"and we once again at a final...", seems to be missing a word. Should it be end or end up at ?